src/shared/condition.c

```diff
@@ -1402,6 +1402,12 @@ ConditionType condition_type_from_string(const char *s) {
         return _condition_type_from_string(s);
 }
 
+void condition_types_list(void) {
+        FOREACH_ARRAY(field, _condition_type_table, ELEMENTSOF(_condition_type_table))
+                puts(*field);
+        puts("ConditionKernelVersion");
```

Comment:
this one is obsolete, i'd hide it?

Reply:
Oh, oh, oh. c154bb65ad55a98fdf7d05fcb6b68a22408ea33a added ConditionVersion=, but didn't hook it up here in bus-unit-util.c at all :(

/cc @teknoraver

Reply:
Ah, no, it works. Phew.